docs/Guide/application-commands/application-command-registry/registering-chat-input-commands.mdx

```diff
@@ -3,56 +3,174 @@ sidebar_position: 3
 title: Registering Chat Input Commands
 ---
 
-A chat input command can be registered to `Discord API` by letting the command registry know about its existence. To do
-so we need to `acquire` or use the `registerApplicationCommands` to have to `ApplicationCommandRegistry` to perform
-operations on it. Let's discuss about `ApplicationCommandRegistry` in more detail.
+A chat input command can be registered to the Discord API by letting the command registry know about its existence. To
+do so we need to acquire or use the [`registerApplicationCommands`][rac] to have access to the
+[`ApplicationCommandRegistry`][acr] to perform operations on it. You can learn how to do this on the page [Acquiring an
+Application Command Registry][aaacr].
+
+## How to register a chat input command
+
+The easiest way to register a Chat Input Command is to implement the [`registerApplicationCommands`][rac] method on a
+[`Command`][command] class. In that method the first parameter is the [`ApplicationCommandRegistry`][acr] (commonly
+called `registry` in code) on which you can call [`registerChatInputCommand`][rcic]. This looks like:
+
+```ts ts2esm2cjs|{7,11-17}|{7,11-17}
```

Comment:
Is the duplicated highlights needed?

Reply:
Yeah I never added copying JS highlights to TS when the latter is omitted.